packages/gatsby-theme-blog/src/components/seo.js

```diff
@@ -19,13 +19,17 @@ function SEO({ description, lang, meta, keywords = [], title }) {
             title
             description
             author
+            siteURL
           }
         }
       }
     `
   )
 
   const metaDescription = description || site.siteMetadata.description
+  const image = imageSource
+    ? `${site.siteMetadata.siteURL}${imageSource}`
```

Comment:
Handy! Sorta dumb question: does this approach account for pathPrefix / assetPrefix?

Reply:
Not dumb. My instinct is that it doesn't and shouldn't need to? But that's a good point and I should test it out.

Reply:
Looked into this more. From what I can see querying it's not necessary to account for it specifically.